src/json.ts

```diff
@@ -60,7 +60,10 @@ export class ValueNode<T extends ValueType = ValueType>
       return this.items[0].value as T;
     }
 
-    return this.items.map((i) => i.value).join("");
+    return this.items
```

Comment:
Nitpick but any reason not to use `.reduce` here as opposed to a map+filter?

Reply:
No real reason -- either way works